tests/plugins/tabletools/manual/cellheight.md

```diff
@@ -0,0 +1,34 @@
+@bender-tags: 4.12.0, 2084, 3098
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, table, tabletools, undo
+
+1. Open the cell properties dialog.
+
+	#### Expected:
+
+	Cell height unit is changeable (pixels or percents).
```

Comment:
I suggest using lists when there are multiple expected.
```suggestion
	- Cell height unit is changeable (pixels or percents).
```

Also
```suggestion
	- Cell height unit is changeable (pixels or percentages).
```
I'm giving two suggestions, as I'm not fully sure about the second one. I think that `percent` means a symbol `%`, while `percentages` is an actual value `33%`.